tests/unit/test_charm.py

```diff
@@ -9,9 +9,16 @@
 from ops import testing
 from ops.model import ActiveStatus, BlockedStatus, WaitingStatus
 from ops.pebble import ExecError
+from sample_nad_defitions import modified_nad_descriptions, original_nad_descriptions
 
 from charm import UPFOperatorCharm
 
+MULTUS_LIBRARY_PATH = "charms.kubernetes_charm_libraries.v0.multus"
+TOO_BIG_MTU_SIZE = 80000
+TOO_SMALL_MTU_SIZE = 0
+VALID_MTU_SIZE_1 = 9000
+VALID_MTU_SIZE_2 = 1800
+
 
```

Comment:
It would be worth to test edge cases : 0, 1200, 1201, 65636, 65637, ...

Reply:
OK, I will expand the tests to include them.

Reply:
Done